shared/timer/Timer.hpp

```diff
@@ -10,12 +10,13 @@ class Timer {
     void start();
     void stop();
     double getTime() const;
-    void resetTime();
+    void reset();
 
 private:
     std::chrono::time_point<std::chrono::high_resolution_clock> startTime;
```

Comment:
add underscore suffix "_" in every private member variable. For example:
`std::chrono::time_point<std::chrono::high_resolution_clock> start_time_;`